CarRental/CarRental.Web/Views/Home/Index.cshtml

```diff
@@ -13,11 +13,16 @@
                 <h1>Car Rental – Search, Compare & Save</h1>
                 <h2>Free cancellations on most bookings 60,000+ locations Customer support in 40+ languages</h2>
                 <form class="d-flex pt-3" asp-controller="Search" asp-action="SearchFromHome" method="post">
-                    <input class="form-control shadow-sm me-2 search opacity-75  " asp-for="SearchDto.ModelAndMake" type="text" placeholder="Search" aria-label="Search">
-                    <input class="form-control shadow-sm me-2 year opacity-75" asp-for="SearchDto.ProductionYearFrom" placeholder="2023" />
-                    <input class="form-control shadow-sm me-2 year opacity-75" asp-for="SearchDto.ProductionYearTo" placeholder="2015" />
-                    <input type="date" class="form-control shadow-sm me-2 date opacity-75" asp-for="SearchDto.StartDate" />
-                    <input type="date" class="form-control shadow-sm me-2 date opacity-75" asp-for="SearchDto.EndDate" />
+                    <input class="form-control shadow-sm me-2 search opacity-75  " asp-for="SearchDto.ModelAndMake" type="text" placeholder="Type car maker or car model" aria-label="Search">
+                    <select asp-for="SearchDto.ProductionYearFrom" id="" name="Year" class="form-control year shadow-sm opacity-75 me-2">
+                        <partial name="_AvailableOptionsForYearFrom" , new { Year=-1 } />
```

Comment:
niepotrzebna duplikacja partiali, po co osobne na To i From ?

Reply:
dunno kopiowalem tego co bylo crobiony do create i w 22 linicje odnosil sie do property Year, z racji tego ze Model ma property YearFrom i YearTo to zrobilem dwa partiale

Reply:
to jest zrobione nie patrzyles nowych commitow